src/vs/editor/common/cursor/cursorTypeEditOperations.ts

```diff
@@ -262,6 +263,21 @@ export class AutoClosingOpenCharTypeOperation {
 			if (!pair.shouldAutoClose(scopedLineTokens, beforeColumn - scopedLineTokens.firstCharOffset)) {
 				return null;
 			}
+
+			// Check if the cursor is inside a comment - if so, don't auto-close comment pairs
```

Comment:
This code should only execute when the auto closing pair is for block comments. Please have a look at line 209 for how to check if the auto-closing pair is a block comment auto closing pair. With the current code, auto closing does not work at all inside of block comments.

Reply:
https://github.com/user-attachments/assets/9cec2137-abd5-4072-984d-17fa9a675163